include/motor-control/core/linear_motion_system.hpp

```diff
@@ -38,6 +40,14 @@ struct LinearMotionSystemConfig {
         return (mech_config.get_mm_per_rev()) /
                (steps_per_rev * microstep * gear_ratio) * 1000;
     }
+    [[nodiscard]] constexpr auto get_encoder_pulses_per_mm() const -> float {
+        return (encoder_ppr * 4.0 * gear_ratio) /
+               (mech_config.get_mm_per_rev());
+    }
+    [[nodiscard]] constexpr auto get_encoder_um_per_pulse() const -> float {
+        return (mech_config.get_mm_per_rev()) /
+               (encoder_ppr * 4.0 * gear_ratio) * 1000.0;
```

Comment:
You might want to return a different value for when the `encoder_ppr` value is 0 here